### ISSUE TITLE
MKS Robin2 -Mesh bed leveling not executing when printing

### ISSUE THREAD

**Issue by DaVincIan** [makerbase-mks/MKS-Robin#105]
My X-Y core printer, with a BL Touch probe is configured for Mesh bed leveling:

#============================= Z Probe Options =============================
>BLTOUCH			1			# 0:disable BLTOUCH; 1:enable BLTOUCH
#Select for a probe connected to Z-Min or Z-Max.
>Z_MIN_PROBE_PIN_MODE		1	# 0 : NULL; 1: ZMIN; 2: ZMAX

>Z_PROBE_OFFSET_FROM_EXTRUDER	-1.35	# Z offset: -below +above  [the nozzle]-0.95;-1.45;-1.2
>X_PROBE_OFFSET_FROM_EXTRUDER	32	# X offset: -left  +right  [of the nozzle]
>Y_PROBE_OFFSET_FROM_EXTRUDER	0	# Y offset: -front +behind [the nozzle]
>XY_PROBE_SPEED 		4000	# X and Y axis travel speed (mm/m) between probes
>Z_PROBE_SPEED_FAST 		600	# Speed for the first approach when double-probing (with PROBE_DOUBLE_TOUCH)
>Z_PROBE_SPEED_SLOW 		300	# Speed for the "accurate" probe of each point

#=============================== Bed Leveling ==============================

>BED_LEVELING_METHOD 		5	# 0:NULL_BED_LEVELING; 3:AUTO_BED_LEVELING_BILINEAR; 5:MESH_BED_LEVELING	

>GRID_MAX_POINTS_X 		9	# 8 the number of grid points per dimension. <= 15
>GRID_MAX_POINTS_Y 		9	# 8 the number of grid points per dimension. <= 15	
>Z_CLEARANCE_DEPLOY_PROBE	10	# Z Clearance for Deploy/Stow	   > 0
>Z_CLEARANCE_BETWEEN_PROBES	5	# Z Clearance between probe points  > 0

# Set the boundaries for probing (where the probe can reach).
>LEFT_PROBE_BED_POSITION 	35	
>RIGHT_PROBE_BED_POSITION 	390
>FRONT_PROBE_BED_POSITION 	0	
>BACK_PROBE_BED_POSITION 	390

>MESH_INSET			35	# Mesh inset margin on print area for MESH_BED_LEVELING

The procedure to perform/gather the bed leveling data performs as expected. It is stored into EEPROM (M500), and I can view/confirm the bed leveling data matrix and status (G29 S0).

**I'm using Repetier Host, with Robin2 V1.0.2 firmware**

The problem is when I print, no bed leveling/correction is taking place. As an exaggerated example:
**Bed leveling disabled:**
Z at X3 Y3 = 0
Z at X200 Y200 = 1

**Bed leveling activated:**
Z at X3 Y3 = 0
Z at X200 Y200 = 1 when Z should be at X200 Y200 = 0 (Obviously I've given the ideal theoretical example)

To summarize, my Z error with bed leveling active and not active is the same.

**Comment by DaVincIan** [comment]
Here is my mesh data:

18:09:29.845 : Measured points:
18:09:29.853 : 0        1        2        3        4        5        6        7        8
18:09:29.862 : 0 -0.31875 -0.33875 -0.36375 -0.38375 -0.42875 -0.47250 -0.49375 -0.47250 -0.41375
18:09:29.867 : 1 -0.28500 -0.30375 -0.30625 -0.34750 -0.37750 -0.42500 -0.44000 -0.43375 -0.36625
18:09:29.876 : 2 +0.01125 -0.00375 -0.03000 -0.04375 -0.05875 -0.12875 -0.17625 -0.19625 -0.14875
18:09:29.881 : 3 +0.11500 +0.08625 +0.05875 +0.02146 +0.00375 -0.08125 -0.08601 -0.08875 -0.05125
18:09:29.889 : 4 +0.00125 -0.02750 -0.03000 -0.04750 -0.10750 -0.15000 -0.18250 -0.17500 -0.10125
18:09:29.898 : 5 -0.09375 -0.09750 -0.09250 -0.14875 -0.16375 -0.24750 -0.26625 -0.28500 -0.24500
18:09:29.903 : 6 -0.12125 -0.15875 -0.17250 -0.17125 -0.23625 -0.29250 -0.31875 -0.35250 -0.32750
18:09:29.912 : 7 -0.19500 -0.24500 -0.27125 -0.30250 -0.35182 -0.37125 -0.44750 -0.44759 -0.45250
18:09:29.920 : 8 -0.11875 -0.23625 -0.28625 -0.30375 -0.33125 -0.38500 -0.44750 -0.50125 -0.46125
18:09:29.925 : X:0.00 Y:0.00 Z:0.00 E:0.00 Count X:0 B:0 Z:0
18:09:30.059 : echo:Settings Stored (1140 bytes; crc 16877)
